### PR TITLE
Update userguide_configuration.adoc

### PR DESCRIPTION
**Pull request by sbrossie**

### CODE REVIEW

File: userguide/platform/userguide_configuration.adoc [hunk 1]
```diff
@@ -29,11 +29,25 @@ boolean isInvoicingSystemEnabled();
 
 == Per tenant properties
 
+Some of our configuration properties can be specified at the tenant level, i.e they can differ for each tenant that specified its own properties. Such properties will have an `InternalTenantContext` in the signature of the method. Example:
```
Comment (pierre) [inline userguide/platform/userguide_configuration.adoc:32]: 👍 but I'm also wondering if we should instead have a table and just list / explain them all (that way, users don't have to go back to the code)?

File: userguide/tutorials/user_management.adoc [hunk 2]
```diff
@@ -27,18 +19,26 @@ Shiro supports multiple backend implementations, and we integrated a few of them
...
 
-Changing such user (e.g to use `superadmin`) or password would require editing `shiro.ini` on the Kill Bill side to make sure this initial user can log-in to Kaui (see below).
+Changing such user (e.g to use `superadmin`) or its password to something else than `password` would require editing `shiro.ini` configuration file on the Kill Bill side to make sure this initial user can log-in to Kaui (see below).
+
+For production deployments, we recommend not using the default `admin` (and its default password `password`).
+
+=== Tenant Access
+
+Note that the authentication/authorization is orthogonal to the tenant access: The set of permissions associated to a given user is the same for all tenants (for which he has the keys). So, a potential superuser with all the permissions and with access to the `api_key`/`api_secret` for all tenants would have full control of the entire system. It is common practise to create a cross-tenant admin with the ability to only manage tenants and nothing else, and then have one superadmin per tenant.
```
Comment (pierre) [inline userguide/tutorials/user_management.adoc]: `practice`

**Review by pierre — APPROVED** [review]

File: userguide/platform/userguide_configuration.adoc (hunk 1 above)
Comment (sbrossie) [reply userguide/platform/userguide_configuration.adoc:32]: Yep, i agree...

File: userguide/tutorials/user_management.adoc (hunk 2 above)
Comment (sbrossie) [reply userguide/tutorials/user_management.adoc]: ack will fix

**Pull request merged**